benchmarks/src/jmh/java/io/grpc/benchmarks/netty/AbstractBenchmark.java

```diff
@@ -369,8 +381,15 @@ public void onComplete() {
 
                       @Override
                       public void onReady() {
-                        while (call.isReady()) {
-                          call.sendMessage(response.slice());
+                        // Cork while the transport declares that it can receive more messages
+                        // so that flushes are deferred until the framer buffer is full.
+                        call.cork();
```

Comment:
This is definitely in the 'go with something' bucket.  Avoiding a object construction for what may end up being a high-frequency event does seem like a benefit here perhaps.